temba/flows/models.py

```diff
@@ -5166,7 +5167,7 @@ def evaluate(self, run, sms, context, text):
 
         state = org.parse_location(text, 1)
         if state:
-            return 1, state
+            return 1, state.first()
```

Comment:
Same here, did the return value for parse_location change and I didn't see it? Isn't it just an admin boundary or None? (on master as far as I can tell parse_location returns one or the other, not a QuerySet)